gtfs_loader/__init__.py

```diff
@@ -224,6 +225,9 @@ def patch(gtfs, gtfs_in_dir, gtfs_out_dir, sorted_output=False):
     for file_schema in schema.GTFS_SUBSET_SCHEMA.values():
         print(f'Writing {file_schema.name}')
         entities = gtfs.get(file_schema.name)
+        if not entities:
+            delete_file(file_schema, gtfs_out_dir)
```

Comment:
```suggestion
            (gtfs_out_dir / file_schema.filename).unlink(missing_ok=True)
```

Python can do this whole operation for you in one go.

Reply:
What? Python preventing me from using `try..catch` pattern?